src/lib/components/Banner.svelte

```diff
@@ -0,0 +1,52 @@
+<script>
+	export let message = 'Welcome to our website!'
+	export let bgColor = '#4CAF50' // Default background color
+
+	let visible = true // Initially, the banner is visible
+</script>
+
+{#if visible}
+	<div class="banner" style="--bg-color: {bgColor};">
+		<span>{@html message}</span>
+		<button class="close-btn" on:click={() => (visible = false)}>×</button>
+	</div>
+{/if}
+
+<style>
+	.banner {
+		padding: 16px;
+		background-color: var(--bg-color);
+		color: var(--black);
+		font-family: var(--font-heading);
+		font-weight: bold;
+		text-align: center;
+		font-size: 1.25em;
+		position: relative;
+	}
+
+	:global(.banner a) {
+		font-family: var(--font-heading);
+		/* font-weight: 700; */
+		color: var(--white);
+	}
+
+	:global(.banner a.active) {
+		color: var(--black);
+	}
+
+	:global(.banner a:hover) {
+		color: var(--black);
+		text-decoration: underline;
+	}
+
+	.close-btn {
+		position: absolute;
```

Comment:
The close button overlaps the text on some screen sizes. You'll probably have an easier time positioning things using flex. Try setting `display: flex` on `.banner` and `flex-grow: 1` on `span`.
![banner](https://github.com/user-attachments/assets/01b4e897-c3ae-47bb-9b92-d0a3779b409a)

Reply:
done. Aligned it vertically on center, it didn't look good if aligned on top